sdk/core/azure-core-http-netty/src/test/java/com/azure/core/http/netty/implementation/RestProxyWithNettyTests.java

```diff
@@ -4,12 +4,36 @@
 package com.azure.core.http.netty.implementation;
 
 import com.azure.core.http.HttpClient;
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder;
+import com.azure.core.test.RestProxyTestsWireMockServer;
 import com.azure.core.test.implementation.RestProxyTests;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 
 public class RestProxyWithNettyTests extends RestProxyTests {
+    private static WireMockServer server;
+
+    @BeforeAll
+    public static void getWireMockServer() {
+        server = RestProxyTestsWireMockServer.getRestProxyTestsServer();
+        server.start();
+    }
+
+    @AfterAll
+    public static void shutdownWireMockServer() {
+        if (server != null) {
+            server.shutdown();
+        }
+    }
+
+    @Override
+    protected int getWireMockPort() {
+        return server.port();
+    }
```

Comment:
This is the same code in `RestProxyWithHttpProxyNettyTests`. Can you check if this can be de-duplicated?

Reply:
For now I will be keep these separate as I believe WireMock is capable of allowing proxying which we may want to take advantage of to final begin testing proxying.